### ISSUE TITLE
`subliminal cache --clear-subliminal' crashes with FileNotFoundError: [Errno 2] No such file or directory: '[...]/.cache/subliminal/subliminal.dbm'

### ISSUE THREAD

**Issue by Coacher** [Diaoul/subliminal#670]
Hello.

subliminal-2.0.2, Python 3.4.

Steps to reproduce:
```
$ ls -la ~/.cache/subliminal/
ls: cannot access /home/coacher/.cache/subliminal/: No such file or directory
$ subliminal cache --clear-subliminal
Traceback (most recent call last):
  File "/usr/lib/python-exec/python3.4/subliminal", line 9, in <module>
    load_entry_point('subliminal==2.0.2', 'console_scripts', 'subliminal')()
  File "/usr/lib64/python3.4/site-packages/click/core.py", line 716, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib64/python3.4/site-packages/click/core.py", line 696, in main
    rv = self.invoke(ctx)
  File "/usr/lib64/python3.4/site-packages/click/core.py", line 1060, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib64/python3.4/site-packages/click/core.py", line 889, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib64/python3.4/site-packages/click/core.py", line 534, in invoke
    return callback(*args, **kwargs)
  File "/usr/lib64/python3.4/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/usr/lib64/python3.4/site-packages/subliminal/cli.py", line 266, in cache
    os.remove(os.path.join(ctx.parent.params['cache_dir'], cache_file))
FileNotFoundError: [Errno 2] No such file or directory: '/home/coacher/.cache/subliminal/subliminal.dbm'
```

**Comment by Coacher** [comment]
Same with subliminal-1.1.1.

**Comment by Coacher** [comment]
And even after cache is created:
```
$ ls -la ~/.cache/subliminal/
total 12
drwxr-xr-x  2 coacher coacher  81 Jun  8 20:37 ./
drwx------ 11 coacher coacher 148 Jun  8 20:37 ../
-rw-r--r--  1 coacher coacher 115 Jun  8 20:37 subliminal.dbm.bak
-rw-r--r--  1 coacher coacher 589 Jun  8 20:37 subliminal.dbm.dat
-rw-r--r--  1 coacher coacher 115 Jun  8 20:37 subliminal.dbm.dir
$ subliminal cache --clear-subliminal
Traceback (most recent call last):
  File "/usr/lib/python-exec/python3.4/subliminal", line 9, in <module>
    load_entry_point('subliminal==2.0.2', 'console_scripts', 'subliminal')()
  File "/usr/lib64/python3.4/site-packages/click/core.py", line 716, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib64/python3.4/site-packages/click/core.py", line 696, in main
    rv = self.invoke(ctx)
  File "/usr/lib64/python3.4/site-packages/click/core.py", line 1060, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib64/python3.4/site-packages/click/core.py", line 889, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib64/python3.4/site-packages/click/core.py", line 534, in invoke
    return callback(*args, **kwargs)
  File "/usr/lib64/python3.4/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/usr/lib64/python3.4/site-packages/subliminal/cli.py", line 266, in cache
    os.remove(os.path.join(ctx.parent.params['cache_dir'], cache_file))
FileNotFoundError: [Errno 2] No such file or directory: '/home/coacher/.cache/subliminal/subliminal.dbm'
```

**Comment by Coacher** [comment]
Please fix.

**Comment by Coacher** [comment]
I guess this is the origin of the problem: http://dogpilecache.readthedocs.io/en/latest/api.html#dogpile.cache.backends.file.DBMBackend.params.filename

**Comment by Coacher** [comment]
Simple fix:
```
diff --git a/subliminal/cli.py b/subliminal/cli.py
index 0b8028d..29249a8 100644
--- a/subliminal/cli.py
+++ b/subliminal/cli.py
@@ -263,7 +263,9 @@ def subliminal(ctx, addic7ed, legendastv, opensubtitles, subscenter, cache_dir,
 def cache(ctx, clear_subliminal):
     """Cache management."""
     if clear_subliminal:
-        os.remove(os.path.join(ctx.parent.params['cache_dir'], cache_file))
+        from glob import glob
+        for file in glob(os.path.join(ctx.parent.params['cache_dir'], cache_file) + '*'):
+            os.remove(file)
         click.echo('Subliminal\'s cache cleared.')
     else:
         click.echo('Nothing done.')
```

**Comment by Diaoul** [comment]
Yep I saw that when working on #663 using the official python:3 image. I prefer to retrieve the real name of the file rather than relying on glob. Need to investigate if there is a way to do that.

**Comment by Coacher** [comment]
I can confirm this is fixed in 2.0.3. Thank you very much.

Please note that `subliminal cache --clear-subliminal` creates `~/.cache/subliminal/` directory even if it doesn't exist.
